README.md

```diff
@@ -1,12 +1,34 @@
-# Bjerk.io
```

Comment:
Maybe this can be nicer than the default Nextjs? Especially since this is a FOSS project.